pkg/controller/pxcrestore/controller.go

```diff
@@ -224,6 +230,25 @@ func (r *ReconcilePerconaXtraDBClusterRestore) getBackup(cr *api.PerconaXtraDBCl
 			},
 		}, nil
 	}
+	if cr.Spec.PITR != nil && cr.Spec.PITR.BackupSource != nil {
```

Comment:
if `cr.Spec.BackupSource` and `cr.Spec.PITR.BackupSource` both defined (and they both must be defined always) this code will never be reached.

Reply:
they shouldn't both be defined Restoring from `cr.Spec.BackupSource` is another functionality, so user should choose the way of restoring

Reply:
removed https://github.com/percona/percona-xtradb-cluster-operator/pull/673/commits/e32dc99092f74f4a3a93b9648e3cbde200ccf9c0

Reply:
I believe we should revert the change and move this block several lines above